### DISCUSSION TITLE
Determining steady state in pipeflow example

### DISCUSSION THREAD

**Discussion by cerice1995** [UvaCsl/HemoCell#63] [General discussion]
I know the time steps needed to reach equilibrium depend on the channel size and Re number, but is there a quick way to check if a simulation has reached steady-state? 

To begin to determine the necessary length of my runs studying steady-state margination, I ran a pipeflow simulation with a 50 um diameter and Re = 0.5 for 4.5 s (45,000,000 1e-7 timesteps with csv's every 1,000,000 timesteps), and each csv snapshot had a significantly different distribution of RBCs (no steady state was apparent). Since this run took several days on 216 CPUs, I wanted to evaluate how much longer I will need to run the simulation before moving forwards with various conditions. 

Also, is there documentation for restarting from a checkpoint? I didn't see it when I searched the site, but I thought I might have read about it before.

**Comment by gzavo** [comment]
Hi @cerice1995,

- At D=50 um diameter with H=20, and Re=0.5 the simulation should reach steady state around 0.1 s if everything is set up correctly.
- There is documentation for checkpointing, you can just use the "Quick search" box on the left bar to locate it.
